util/regexp-assemble/data/933160.data

```diff
@@ -1,9 +1,25 @@
 ##! Please refer to the documentation at 
 ##! https://coreruleset.org/docs/development/regexp_assemble/.
+##! \b
```

Comment:
Remove these commented bits.

Reply:
Previous commented prefix and suffix removed.